### PR TITLE
Test and structure

### PR DESCRIPTION
**Pull request by AAM77**
- Added /api/v1 name spacing to routes
- Moved controllers into appropriate directory
- Added basic seed code to seeds.rb
- Removed password attribute from serializer
- Updated dev-log

- Next step: add tests for the api to test routes, the existence of controllers, validations, and whether the existing controllers and actions are behaving as intended.
- (First) #=> Add basic tests for existing routes and controllers

### CODE REVIEW

File: app/serializers/user_serializer.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class UserSerializer < ActiveModel::Serializer
-  attributes :id, :username, :email, :password
+  attributes :id, :username, :email
```
Comment (achasveachas) [inline app/serializers/user_serializer.rb]: Good catch!

File: db/seeds.rb [hunk 2]
```diff
@@ -6,3 +6,12 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
  User.create(username: 'admin', password: "password")
+
+usernames = ['apple', 'orange', 'banana', 'pear']
```
Comment (achasveachas) [inline db/seeds.rb]: The [faker](https://github.com/faker-ruby/faker) gem is a great resource for seed/test data!

File: app/serializers/user_serializer.rb (hunk 1 above)
Comment (CodeToJoyForever) [reply app/serializers/user_serializer.rb]: Thank you!

File: db/seeds.rb (hunk 2 above)
Comment (CodeToJoyForever) [reply db/seeds.rb]: Oh yes, that's right! I meant to add that. Good catch & thank you!

**Comment by CodeToJoyForever** [comment]
Closing this pull request.

**Pull request merged**